api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -420,6 +422,7 @@ private User initializeUserIfNeeded() {
         request.getProfile().getOrganization(),
         request.getProfile().getAreaOfResearch()
     );
+    validateProfileFields(user);
```

Comment:
I think this should be an easy change:

- Switch validation from `User` -> `Profile`
- Move the validation ahead of the user creation

This would be nice because it would prevent creation of an invalid user.